backstory/src/main/webapp/style.css

```diff
@@ -278,6 +277,68 @@ textarea {
   font-weight: bold;
 }
 
+/* Vision Demo Page */
+#upload-form-div {
+  align-items: center;
+  width: 80%;
+}
+
+#image-upload-form {
+  text-align: center;
+  width: 100%;
+}
+
+/** Make the image upload input invisible */
+.file-input {
```

Comment:
Looks like this was already declared above.

Reply:
Will remove this. Thank you!

Reply:
Done